pkg/cmd/cli/cmd/logs.go

```diff
@@ -163,6 +168,13 @@ func (o *OpenShiftLogsOptions) RunLog() error {
 		}
 		return o.runLogsForBuild(build)
 
+	case "dc", "deploymentconfig", "deploymentconfigs":
```

Comment:
I'm not sure what comment you're pointing to - for actually resolving names
we should always use RESTMapper full stop.  You can round trip RESTMapper
back to the full type if you somehow need that.

On Oct 16, 2015, at 10:47 AM, Michail Kargakis <notifications@github.com>
wrote:

In pkg/cmd/cli/cmd/logs.go
<https://github.com/openshift/origin/pull/3943#discussion_r42250620>:

> @@ -163,6 +168,13 @@ func (o *OpenShiftLogsOptions) RunLog() error {
>  		}
>  		return o.runLogsForBuild(build)
>
> +	case "dc", "deploymentconfig", "deploymentconfigs":

We have that in the RESTMapper already - why do we need a second location.

See #4947 (comment)
<https://github.com/openshift/origin/pull/4947#discussion_r41714595>

does VersionAndKindForResource handle singlar resources

Yes, it does.

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/3943/files#r42250620>.